doc/sphinx/practical-tools/coqide.rst

```diff
@@ -5,130 +5,183 @@
 Coq Integrated Development Environment
 ========================================
 
-The Coq Integrated Development Environment is a graphical tool, to be
-used as a user-friendly replacement to `coqtop`. Its main purpose is to
-allow the user to navigate forward and backward into a Coq
-file, executing corresponding commands or undoing them respectively.
-
-CoqIDE is run by typing the command `coqide` on the command line.
-Without argument, the main screen is displayed with an “unnamed
-buffer”, and with a filename as argument, another buffer displaying
-the contents of that file. Additionally, `coqide` accepts the same
-options as `coqtop`, given in :ref:`thecoqcommands`, the ones having obviously
-no meaning for CoqIDE being ignored.
+.. todo PR: how to say that a number of things are broken?  Maybe list them
+   somewhere--doesn't have to be super detailed
+
+The Coq Integrated Development Environment (CoqIDE) is a user-friendly GUI
+for Coq. Its main purpose is to
+allow users to edit scripts and step forward and backward through Coq scripts,
+respectively executing and undoing commands and tactics.
+
+To run CoqIDE, enter `coqide` on the command line.
+If you include script file names as arguments, each is opened
+in a separate tab.  If you don't, CoqIDE opens a single unnamed buffer
+(titled `*scratch*`).  `coqide` also accepts many of the options of `coqtop`
+(see :ref:`thecoqcommands`), while ignoring the ones that aren't meaningful
+for CoqIDE.  Use `coqide --help` to see the list of command line options.
 
 .. _coqide_mainscreen:
 
   .. image:: ../_static/coqide.png
      :alt: CoqIDE main screen
 
-A sample CoqIDE main screen, while navigating into a file `Fermat.v`,
-is shown in the figure :ref:`CoqIDE main screen <coqide_mainscreen>`.
-At the top is a menu bar, and a tool bar
-below it. The large window on the left is displaying the various
-*script buffers*. The upper right window is the *goal window*, where
-goals to be proven are displayed. The lower right window is the *message
-window*, where various messages resulting from commands are displayed.
-At the bottom is the status bar.
+The screen shot shows CoqIDE while the user is stepping through the file `Fermat.v`.
+A menu bar and tool bar appear at the top of the window. The left-hand panel shows
```

Comment:
```suggestion
A menu bar and a tool bar appear at the top of the window. The left-hand panel shows
```
I'm not sure about this one however. But I'll state it in case you agree.